programs/vote/src/vote_instruction.rs

```diff
@@ -20,6 +20,7 @@ use solana_sdk::{
 use thiserror::Error;
 
 /// Reasons the stake might have had an error
+#[frozen_abi(digest = "ByLc61HbxwBPJ54fQZQFmjyzD4qywr5MuhrN9eHhgsT")]
```

Comment:
Alternative names: `checksum`, `hash`, `version`. No particular preference :)

Reply:
Also, diverting from the book, I chose base58 over hex, purely because of concise and simple to implement.

Reply:
digest seems fine with me, as does base58 since we use it so much elsewhere